lib/diagnostic.js

```diff
@@ -33,6 +48,23 @@ db.dropDatabase();
 
 // your code begins here
 
+db.house.update(
```

Comment:
`each` should be `$each`.  The key should also be named `members`, not `member`.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```